components/RichTextEditor.js

```diff
@@ -286,6 +287,57 @@ export default class RichTextEditor extends React.Component {
     }
   }
 
+  /** A helper used by autolink to find where the url actually ends */
+  autolink_delim = url => {
```

Comment:
`autolink_delim` should be pascalCase. We also prefer avoiding abbreviation like "delim" and use complete words like "delimiter".